widget/edit.controller.js

```diff
@@ -58,6 +71,21 @@
             $scope.listItems = data.options;
          });
       };
+
+      function loadAttributestoupdate() {
+        $scope.newfieldsArray = [];
+        var entity = new Entity($state.params.module);
+        entity.loadFields().then(function () {
+          $scope.newfieldsArray = entity.getFormFieldsArray();
+          $scope.newfieldsArray = _.filter($scope.newfieldsArray, function(item){ return item.model === 'picklists'; });
+          if(config.newpicklistItem){
+            var selectedPicklistItem = _.find($scope.newfieldsArray, function(item){
+            return item.name === config.newpicklistItem;
```

Comment:
$scope.config.newpicklistItem will be used. please verify here as well

Reply:
updated